pagopa/pn-frontend
# Pg login from selfcare portal - pn-5157

## Short description
Now the login for pg is managed by selfcare portal and not by login webapp

## List of changes proposed in this pull request
- Disabled PG management in login webapp
- Removed type query param when pf webapp redirects to login webapp
- Managed new login and aar in pg webapp

## How to test
- PF -> login with pf and check that the login works correctly
- PF -> use an aar link and check that it works correctly both for not logged user and logged one
- Login -> check that disambiguation page is never shown
- PG -> login with pg and check that the login works correctly
- PG -> logout with pg and check that we are redirected to selfcare portal and not login webapp
- PG -> use an aar link and check that it works correctly both for not logged user and logged one
